src/csv_overrides.py

```diff
@@ -274,7 +274,8 @@ def read_file(
             csv_error(path, i, "Duplicate identifier", value)
             continue
 
-        result[key] = value
+        for k in key.split("|"):
+            result[k.strip()] = value
```

Comment:
I'm a bit surprised that this code works.  I would think that in https://github.com/pokey/cursorless-talon/blob/e75cda037cca8a2de734027446234893f1661143/src/csv_overrides.py#L129 we would just end up with whichever key appears last.  Presumably you've checked that this code works?  Any idea why it doesn't get overridden in that line?

Man we really need tests for this stuff 😅

Reply:
You're probably right I need to test some more.

Reply:
Now it's testing and working